src/Liquifier.sol

```diff
@@ -89,6 +89,8 @@ contract Liquifier is Initializable, UUPSUpgradeable, OwnableUpgradeable, Pausab
 
     mapping(address => bool) public DEPRECATED_pausers;
 
+    address etherFiRestakeManager;
```

Comment:
how do you initialize `etherFiRestakeManager`? 

https://github.com/etherfi-protocol/smart-contracts/pull/190/files#diff-3580438b4fe6983efe4f083d8f61ac9ee103fedfbfb9731cee861e85f6c4351dR138-R140

Reply:
`etherFiRestakeManager` uses the same storage slot as the already initialized `etherfiRestaker` variable, so I am essentially just renaming `etherfiRestaker` ->  `etherFiRestakeManager`. While this is not best practice, I want to avoid creating another DEPRECATED variable.

Reply:
Do we want to handle the upgrade to the Restake Manager here?